src/backend/base/langflow/components/models/maritalk.py

```diff
@@ -24,29 +24,37 @@ class MaritalkModelComponent(LCModelComponent):
             name="model_name",
             display_name="Model Name",
             advanced=False,
-            options=["sabia-2-small", "sabia-2-medium"],
-            value=["sabia-2-small"],
+            info="Choose between Sabiazinho-3 (smaller and faster) or Sabia-3 (larger and more capable) models.",
+            options=["sabiazinho-3", "sabia-3"],
+            value="sabiazinho-3",
```

Comment:
What do you think of allowing users to customize the model?
```suggestion
            info="Choose between Sabiazinho-3 (smaller and faster) or Sabia-3 (larger and more capable) models.",
            options=["sabiazinho-3", "sabia-3"],
            value="sabiazinho-3",
            combobox=True
```

Reply:
Hi @ogabrielluiz , I can do it like Hugging Face too, some models available and a custom field to add as the user wants. Could it be like this? This way it maintains good usability without making it difficult to add other models.

Reply:
https://github.com/user-attachments/assets/396f284d-3d14-4ac5-b8bb-48e9f45c08df


I modified the component, now it has the following functionalities:

- 2 models by default
- If there is an API key, it searches for models through the API and adds them to the dropdown
- It has a custom field that allows the user to add models as desired.

I will do commit the changes.

Reply:
https://github.com/user-attachments/assets/65823cb3-ee99-4d1e-9f7f-7384620cdadf

Thank you, I had never used the combobox, I liked the functionality!

Should I leave it with just the API?
and without models by default? And if you want, you can use additional ones manually.

Like in the attached video?

Reply:
No, we should avoid relying on external systems. Always have a fallback.

Reply:
Thanks, I will modify it!